sources/packages/api/src/route-controllers/models/common.dto.ts

```diff
@@ -76,3 +76,12 @@ export class OptionItemAPIOutDTO {
   id: number;
   description: string;
 }
+
+/**
+ * Allow extending the object properties dynamically
+ * while keeping the type.
+ */
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
do we need this `eslint-disable-next-line`

Reply:
Yes, there is a warning that states that the generic "T" was not used when it is actually defining the type of the dynamic property. It seems a false positive to me.